<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-GEDTK2H" name="Test" tests="1" failures="0" timestamp="13 Jul 2019 02:44:35 GMT" time="93.072" errors="0">
  <testcase name="createOrderTest" time="75.059" classname="serviceHub.OrderManagement.CreateOrder">
    <failure type="atu.testng.reports.exceptions.ATUReporterStepFailedException">
      <![CDATA[[ATU Custom Reporter Step Failed Exception]
	at atu.testng.reports.listeners.ATUReportsListener.onTestSuccess(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- createOrderTest -->
</testsuite> <!-- Test -->
